parlai/core/dict.py

```diff
@@ -71,8 +71,9 @@ class DictionaryAgent(Agent):
     default_maxngram = -1
     default_minfreq = 0
     default_null = '__NULL__'
-    default_eos = '__EOS__'
     default_unk = '__UNK__'
+    default_eos = '__EOS__'
+    default_go = '__GO__'
```

Comment:
Thanks for the addition Henry! Can you change __GO__ to __START__ and __EOS__ to __END__? slightly more generic names I guess / they match each other better